src/commands/games/freefire/weapon.js

```diff
@@ -0,0 +1,66 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../../')
+
+const Fuse = require('fuse.js')
+
+module.exports = class FreeFireWeapon extends Command {
+  constructor (client) {
+    super({
+      name: 'weapon',
+      aliases: ['w'],
+      parent: 'freefire',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:freefire.subcommands.weapon.missingWeapon'
+      }]
+    }, client)
+  }
+
+  async run ({ t, channel, language }, query) {
+    const {
+      weapons,
+      commons: { attribute_names: attributeNames, attachment_names: attachmentNames }
+    } = await this.client.apis.freefire.getWeaponData(language)
+
+    const fuse = new Fuse(weapons, {
+      shouldSort: true,
+      maxPatternLength: 32,
+      minMatchCharLength: 1,
+      keys: [
+        'name'
+      ]
+    })
+
+    const weapon = fuse.search(query)[0]
+
+    if (!weapon) {
+      throw new CommandError(t('commands:freefire.subcommands.weapon.unknownWeapon', {
+        weapon: query
+      }))
+    }
+
+    const embed = new SwitchbladeEmbed()
+      .setTitle(weapon.name)
+      .setDescription(`**${weapon.description}**`)
+      .addFields([
+        {
+          name: t('commands:freefire.subcommands.weapon.attributes'),
+          value: this.getInfo({ t, weapon, data: 'attributes', names: attributeNames }),
+          inline: true
+        },
+        {
+          name: t('commands:freefire.subcommands.weapon.attachments'),
+          value: this.getInfo({ t, weapon, data: 'attachments', names: attachmentNames }),
+          inline: true
+        }
+      ])
+      .setImage(weapon.skins[0].image_url)
+
+    await channel.send(embed)
+  }
+
+  getInfo ({ t, weapon, data, names }) {
+    const locale = 'commands:freefire.subcommands.weapon.answer.'
+    return Object.entries(weapon[data])
+      .map(([key, value]) => ` - **${names[key]}:** ${data === 'attachments' ? value.avaliable ? t(locale + 'yes') : t(locale + 'no') : value}`)
+      .join('\n')
+  }
```

Comment:
Maybe instead of displaying information like

- **Something:** Yes
- **Other thing:** No
- **Third thing:** Yes
- **Fourth thing:** No

We could only show the attachments/attributes that are available, instead. In that case, the above example would look like so, with unavailable things ommited:

- Something
- Third thing